docs/docs/config/displays.md

```diff
@@ -62,3 +62,12 @@ See the Devicetree bindings for your display. Here are the bindings for common d
 - [SSD1306 (spi)](https://docs.zephyrproject.org/3.5.0/build/dts/api/bindings/display/solomon,ssd1306fb-spi.html)
 
 A full list of drivers provided by Zephyr can be found in [Zephyr's Devicetree bindings index](https://docs.zephyrproject.org/3.5.0/build/dts/api/bindings.html).
+
```

Comment:
FYI this is how we normally document these in config pages: https://zmk.dev/docs/config/kscan#devicetree

Reply:
Thanks. Missed that. I'll update.

Reply:
@caksoylar Tweaked.